hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/RpcConnection.java

```diff
@@ -126,8 +136,13 @@ protected RpcConnection(Configuration conf, HashedWheelTimer timeoutTimer, Conne
       LOG.debug("Use " + authMethod + " authentication for service " + remoteId.serviceName
           + ", sasl=" + useSasl);
     }
+
     reloginMaxBackoff = conf.getInt("hbase.security.relogin.maxbackoff", 5000);
     this.remoteId = remoteId;
+
+    forceReloginEnabled = conf.getBoolean("hbase.security.force.relogin.enabled", false);
+    // Default minimum time between force relogin attempts is 10 minutes
+    this.forceReloginBackoff = conf.getInt("hbase.security.force.relogin.backoff", 10 * 60 * 1000);
```

Comment:
See above comment. These config names do not help an operator to know what to do at all.